syntax = "proto3";

package rgen;

message Request {
    string patient_id = 1;
    string link_to_xray = 2;
}

message Response {
    string patient_id = 1;
    string report = 2;
}

service RGen {
    rpc GenerateReport (Request) returns (Response);
}
